src/index.ts

```diff
@@ -52,12 +54,32 @@ process.on('unhandledRejection', err => {
         new Boom('The requested resource was not found.', { statusCode: 404 })
     });
 
-    const pubSubClient = new PubSub({ projectId: 'atb-mobility-platform' });
-    const js = journeyService(enturService, pubSubClient);
+    const pubSub = createPublisher(
+      new PubSub({ projectId }),
+      {
+        getTripPatternsTopic:
+          process.env['PUBSUB_TOPIC_TRIP_SEARCH'] || 'analytics_trip_search',
```

Comment:
Mulig dette burde være egen config-fil for å synliggjøre konfigurasjonsmulighetene.

Reply:
Dette kommer som en del av en terraform-modul hvor disse er spesifisert. Kunne sikkert fått plass i README.